### PR TITLE
added Jeffwan as a wg serving chair

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
As per https://github.com/kubernetes/community/issues/7876

### CODE REVIEW

**Review by terrytangyuan — APPROVED** [review]
Great to have you! @Jeffwan 

/lgtm

File: groups/groups.yaml [hunk 1]
```diff
@@ -153,3 +153,4 @@ groups:
       - xandergrzyw@gmail.com # CoCC
       - xingyang105@gmail.com
       - ynli@google.com
+      - wg-serving-leads@kubernetes.io # WG Serving leads
```
Comment (dims) [inline groups/groups.yaml:156]: So far we have not been doing this. So can we please not add this line here?

Comment (SergeyKanzhelev) [reply groups/groups.yaml:156]: so I should include people one by one? Is there any troubles with group memberships? I saw it being used in some other groups

Comment (SergeyKanzhelev) [reply groups/groups.yaml:156]: Also this list already have an example: 

https://github.com/kubernetes/k8s.io/blob/5c1916e60425471b3889daf9b34518943c02c277/groups/groups.yaml#L51

Comment (dims) [reply groups/groups.yaml:156]: I'll clean that up @SergeyKanzhelev. must have missed that

**Comment by ArangoGutierrez** [comment]
Any update here?

File: groups/groups.yaml (hunk 1 above)
Comment (SergeyKanzhelev) [reply groups/groups.yaml:156]: fixed. Just for my education, what is wrong with aliases in this list?

Comment (ArangoGutierrez) [reply groups/groups.yaml:156]: Hi @dims , can we move forward with this PR?

Comment (dims) [reply groups/groups.yaml:156]: let's please drop `wg-serving-leads@kubernetes.io` from this list.

Comment (SergeyKanzhelev) [reply groups/groups.yaml]: Now actually fixed it. My excuse it that I trusted VSCode UI for PR editing, but it left the change unmerged. Sorry about it

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Comment by Jeffwan** [comment]
@dims do you think we can cancel the hold for the PR now?

**Comment by dims** [comment]
/hold cancel

thanks @SergeyKanzhelev @Jeffwan

**Pull request merged**